packages/allow-scripts/src/types/common.ts

```diff
@@ -1,15 +1,29 @@
+import type { JsonObject, PackageJson } from 'type-fest';
```

Comment:
This wants `type-fest` to now be a prod dependency.

Reply:
What is the assumption behind that importing it here vs in the jsdoc changes is status?

Reply:
for posterity:

Exporting a type containing `JsonObject` means that unless `type-fest` is a production type, downstream users won't know what `JsonObject` is.